docs/organizations/integrations/slack-integration.md

```diff
@@ -0,0 +1,35 @@
+# Organization Slack integration for Security issues
+
+The organization Slack integration for Security issues sends notifications to a Slack channel of your choice whenever Codacy detects new critical Security issues in the default branch of any repository in your organization.
+
+## Installing the Slack integration
+
+To install the Slack integration:
+
+1.  **On Slack**:
+
+    1.  [Create an Incoming Webhook](https://api.slack.com/messaging/webhooks) and set it to post to a channel of your choice.
```

Comment:
It's a good question. I assumed it was different because it's trying to explain the concept of Incoming Webhooks, not starting the flow for integrating. As you said, that flow already happens and is easier to follow from the UI.

```suggestion
    1.  Create an [Incoming Webhook](https://api.slack.com/messaging/webhooks) and set it to post to a channel of your choice.
```

Reply:
cc @Afonsobli 🙏

Reply:
[Updated](https://github.com/codacy/docs/blob/839c875079f8eebdb37f578f152f051159b1d872/docs/organizations/integrations/slack-integration.md?plain=1#L10-L14) 👍